src/schema/nmdc.yaml

```diff
@@ -307,6 +307,21 @@ classes:
     exact_mappings:
       - OBI:0000747
       - NCIT:C43412
+
+  study role:
+    is_a: named thing
+    aliases: ['credit table', 'associated researchers']
+    description: >-
+      This class supports binding associated researchers to studies. There will be at least a slot for a CRediT Contributor Role (https://casrai.org/credit/) and for a person value
+      Specifically see the associated researchers tab on the NMDC_SampleMetadata-V4_CommentsForUpdates at https://docs.google.com/spreadsheets/d/1INlBo5eoqn2efn4H2P2i8rwRBtnbDVTqXrochJEAPko/edit#gid=0 
+    notes: >-
+      Does this require an in_subset assertion?
+      Will each study role instance require an id, since they're named things?
+      A study should be able to take  0..inf study roles
+    slots:
+      - applies to person
+      - has contributor role
+    class_uri: prov:Association
```

Comment:
I believe line 324 says that the uri of `study role` is `prov:Association` when the schema is converted to RDF. See [class_uri](https://linkml.io/linkml-model/docs/class_uri/). Is right @cmungall ?